src/plugins/wizard/public/application/components/experimental_info.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { memo } from 'react';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@osd/i18n/react';
+
+export const InfoComponent = () => {
```

Comment:
did it not work to just export/import https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualize/public/application/components/experimental_vis_info.tsx ?

Reply:
Yeah, we probably just want to update the regular copy, too. But we can do that separately